src/workerd/api/r2-admin.h

```diff
@@ -5,6 +5,7 @@
 #pragma once
 
 #include "r2-bucket.h"
```

Comment:
Not necessarily for this PR but I highlighted this as a file that needs to move into the internal repo as it was exposed accidentally (it's not now nor will it ever be a public-facing API).